components/root/root.tsx

```diff
@@ -301,6 +301,8 @@ export default class Root extends React.PureComponent<Props, State> {
             BrowserStore.setLandingPageSeen(true);
         }
 
+        this.trackUTMCampain();
+
```

Comment:
Can we move this before the client apps landing page above as they may be redirected in the `if` above and we won't reach this track function?

Reply:
Yeah that makes sense, I'll move this call right after rudder is configured